java/app-encryption/README.md

```diff
@@ -14,6 +15,24 @@ Application level envelope encryption SDK for Java with support for cloud-agnost
     * [Handling read\-only Docker containers](#handling-read-only-docker-containers)
   * [Development Notes](#development-notes)
 
+## Installation
+
+You can include Asherah in Java projects projects using [Maven](https://maven.apache.org/)
+
+The Maven group ID is `com.godaddy.asherah`, and the artifact ID is `appencryption`.
+
+You can specify the current release of Asherah as a project dependency using the following configuration:
```

Comment:
Something like this would probably read better: "To add a dependency using Maven, use the following:"